documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,53 +1,26 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} automatically creates a Prometheus `ServiceMonitor` that scrapes metrics from your {brandname} cluster.
 
-.Prerequisites
+.Procedure
 
-* Have an `oc` client.
-* Enable monitoring for user-defined projects on {ocp}.
+Enable monitoring for user-defined projects on {ocp} so that the `ServiceMonitor` resource is available on the {openshiftshort} cluster.
 
-.Procedure
+When it detects the `ServiceMonitor` resource, {ispn_operator} does the following:
```

Comment:
Would it make sense to use rather _When the `ServiceMonitor` resrouce is detected on the cluster, ..._. Calling the Operator _it_ doesn't fits in here as the context is missing